src/settings/LocationLocations/LocationManager.js

```diff
@@ -459,58 +460,50 @@ class LocationManager extends React.Component {
     this.transitionToParams({ layer: 'clone' });
   };
 
+  checkLocationHasHoldingsOrItems = async (locationId) => {
+    const { mutator } = this.props;
+    const query = `permanentLocationId=${locationId} or temporaryLocationId=${locationId}`;
```

Comment:
maybe it should contain double equals
```suggestion
    const query = `permanentLocationId=="${locationId}" or temporaryLocationId=="${locationId}"`;
```